spec/definitions/Scheduling/ExportScheduleInstruction.yaml

```diff
@@ -0,0 +1,15 @@
+description: |
+  The definition of the recurring schedule.
+  If omitted, then the export happens immediately and is not recurring.
+type: object
+discriminator: method
+properties:
+  method:
```

Comment:
this would be better as `frequency` (ref: https://tools.ietf.org/html/rfc5545#section-3.8.5)